src/cli/cli_srp_server.cpp

```diff
@@ -206,6 +293,24 @@ template <> otError SrpServer::Process<Cmd("lease")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli srp server host
+ * @code
+ * srp server host
+ * srp-api-test-1.default.service.arpa.
+ *     deleted: false
+ *     addresses: [fdde:ad00:beef:0:0:ff:fe00:fc10]
+ * srp-api-test-0.default.service.arpa.
+ *     deleted: false
+ *     addresses: [fdde:ad00:beef:0:0:ff:fe00:fc10]
+ * Done
+ * @endcode
+ * @par
+ * Returns information about all registered SRP server hosts.
```

Comment:
Change `registered SRP server hosts` to `registered hosts`? (to avoid confusion that they are registered by server).
```suggestion
 * Returns information about all registered hosts.
```